### PR TITLE
add `Use WorkloadSpread with customized workload` manual

### PR DESCRIPTION
**Pull request by AiRanthem**
add `Use WorkloadSpread with customized workload` manual to 1.6 1.7 1.8

### CODE REVIEW

File: docs/user-manuals/workloadspread.md [hunk 1]
```diff
@@ -204,7 +223,95 @@ The workload managed by WorkloadSpread will scale according to the defined order
 # deletion order: b -> a -> c
 ```
 
+## Use WorkloadSpread with customized workload
+
+If you want to use WorkloadSpread with custom workloads, which is disabled by default, some
```
Comment (zmberg) [inline docs/user-manuals/workloadspread.md:228]: Please explain the customized workload must support scale sub-resources

**Review by furykerry — COMMENTED** [review]
plz update .github/workflows/pre_dict.json to avoid the typo check error

File: docs/user-manuals/workloadspread.md (hunk 1 above)
Comment (AiRanthem) [reply docs/user-manuals/workloadspread.md:209]: scale sub-resources seems not to be used

**Review by furykerry — COMMENTED** [review]
/lgtm

**Review by furykerry — COMMENTED** [review]
/lgtm

**Pull request merged**